static/css/bureau-structure.less

```diff
@@ -103,73 +108,56 @@
         border-color: @gray-20;
         border-style: solid;
         border-width: 1px 0 0 0;
-        
+
         .respond-to-max(599px, {
             border-width: 1px 0 1px 0;
         });
     }
-    
+
     &_node {
         width: 100%;
-        
+
         .respond-to-min(600px, {
             position: relative;
             margin-bottom: unit(@grid_gutter-width / 2, px);
```

Comment:
[`@grid_gutter-width`](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L42) is already in pixels. Looks like divide by 2 doesn't need the `unit()` helper (per https://github.com/cfpb/cfgov-refresh/pull/401/files#diff-6b6af7d41d4b8c5dc14e6baaa8c1cef2R74), but double-check.